physionet-django/user/models.py

```diff
@@ -757,6 +757,12 @@ class CredentialApplication(models.Model):
         (4, 'Revoked')
     )
 
+    class AutoRejectionReason(models.IntegerChoices):
```

Comment:
Can we use the Django enum here for readability (so in the email template, we can call this with a human readable value, instead of an integer?)

Reply:
For sure. let me make an update